mriqc/utils/bids.py

```diff
@@ -32,3 +35,62 @@ def collect_bids_data(
         )
 
     return imaging_data
+
+
+def write_bidsignore(deriv_dir):
+    bids_ignore = (
+        "*.html", "logs/",  # Reports
+        "*_T1w.json", "*_T2w.json", "*_bold.json",  # Outputs are not yet standardized
+    )
+    ignore_file = Path(deriv_dir) / ".bidsignore"
+
+    ignore_file.write_text("\n".join(bids_ignore) + "\n")
+
+
+def write_derivative_description(bids_dir, deriv_dir):
+    from ..__about__ import __version__, __download__
+
+    bids_dir = Path(bids_dir)
+    deriv_dir = Path(deriv_dir)
+    desc = {
+        'Name': 'MRIQC - MRI Quality Control',
+        'BIDSVersion': '1.4.0',
+        'DatasetType': 'derivative',
+        'GeneratedBy': [{
+            'Name': 'MRIQC',
+            'Version': __version__,
+            'CodeURL': __download__,
+        }],
+        'HowToAcknowledge':
+            'Please cite our paper (https://doi.org/10.1371/journal.pone.0184661).',
+    }
+
+    # Keys that can only be set by environment
+    # XXX: This currently has no effect, but is a stand-in to remind us to figure out
+    # how to detect the container
+    if 'MRIQC_DOCKER_TAG' in os.environ:
```

Comment:
the dockerfile does not seem to set this variable

Reply:
Hence the XXX. I copied this from fMRIPrep, which also doesn't define it.

Reply:
ups, just noticed the comment - sorry about that.